func_adl_xAOD/common/event_collections.py

```diff
@@ -79,3 +83,10 @@ def get_collection(self, info, call_node):
     def create_higher_order_function(self, info):
         'Creates a higher-order function because python scoping is broken'
         return lambda call_node: self.get_collection(info, call_node)
+
+    def get_method_names(self):
+        method_names = {}
+        for info in self._collections:
+            method_names[info['function_name']] = self.create_higher_order_function(info)
+        method_names['DeltaR'] = DeltaRAst
```

Comment:
This needs to be close to where the `DelltaRAst` is defined. Perhaps put a function there which is imported and called and then the `method_names.update()` is called with the return.